### DISCUSSION TITLE
View TREL Peers

### DISCUSSION THREAD

**Discussion by fgervais** [openthread/openthread#8676] [Q&A] [answered]
Is there an `ot-ctl` command to view TREL peers?

**Comment by abtink** [comment, marked as answer]
Yes. [`trel peers`](https://github.com/openthread/openthread/blob/30aebb5e74064d63513ea78dd4fa2ecc46fd38f9/src/cli/README.md#trel-peers-list)

```bash
> trel peers
| No  | Ext MAC Address  | Ext PAN Id       | IPv6 Socket Address                              |
+-----+------------------+------------------+--------------------------------------------------+
|   1 | 5e5785ba3a63adb9 | f0d9c001f00d2e43 | [fe80:0:0:0:cc79:2a29:d311:1aea]:9202            |
|   2 | ce792a29d3111aea | dead00beef00cafe | [fe80:0:0:0:5c57:85ba:3a63:adb9]:9203            |
Done
```

You can find documentation for all CLI commands in [`cli/README.md`](https://github.com/openthread/openthread/blob/main/src/cli/README.md#openthread-cli-reference)

**Reply by fgervais** [reply]
Ah makes sense 😅. Thank you for pointing me to the related documentation.
